tests/main/CMakeLists.txt

```diff
@@ -71,3 +71,7 @@ DEFINE_LAUNCH_TEST(distant_regulation)
 DEFINE_LAUNCH_TEST(special_characters)
 
 DEFINE_LAUNCH_TEST(phase_shifter)
+
+DEFINE_LAUNCH_TEST(launch_fail_generators)
+## Expected to fail because no generator regulates
```

Comment:
regulates the voltage in the main connected component

Reply:
OK